README.md

```diff
@@ -1,16 +1,32 @@
+## Pre-requisites
+
+- You will need to have an Azure account already available
+- Azure CLI installed to use this provider.
+- An Azure OpenAI endpoint provisioned with the models you would like to use.
+
+You will need to provide the following information when first setting up the provider:
+
+- Azure Subscription ID
+- Azure Resource Group
+- Azure OpenAI model name
+
 ## Usage Example
 
+To override the default model of all your gptscripts you can run the following command:
```

Comment:
It might be worth linking to the gptscript docs from here, because IIRC this won't override the model used for scripts that specify a model inline. There's potentially a little more to it than this, and there's also the option of setting this as an env var for people who want to use Azure all the time.